trustfall_core/src/frontend/mod.rs

```diff
@@ -1161,7 +1160,7 @@ where
                 let tag_field = ContextField {
                     vertex_id: current_vid,
                     field_name: subfield.name.clone(),
-                    field_type: subfield_raw_type.to_owned(),
+                    field_type: subfield_raw_type.clone(),
```

Comment:
I changed this to be the same as the `.clone()`'s above, but not sure if it's necessary